draft-ietf-teep-architecture.md

```diff
@@ -1329,23 +1329,47 @@ may need to be updated.
 ## Compromised TAM
 
 Device TEEs are responsible for validating the supplied TAM certificates
-to determine that the TAM is trustworthy. A compromised TAM can install
-arbitrary TAs to, potentially, many devices, which makes TAMs high value
-targets. An advanced, persistent threat actor could potentially compromise
-a TAM without impacting its certificate or raising concern from the TAM's
-operator. To mitigate this threat, TEEs have several options for 
-detecting and mitigating a compromised TAM:
+to determine that the TAM is trustworthy. A compromised TAM may bring multiple threats 
+and damage to user devices that it can managee and the device owners. 
+In one hand, the device information of many devices that the TAM manages and 
+therefore have in its system may be leaked to a bad actor. On the other hand,
+a compromised TAM can install many TAs to launch somee DoS attack to devices,
+for example, by filling up a device's TEE resources reserved for TAs such that 
+other TAs may not get resource to be installed or properly function. It may
+also install malicious TAs to, potentially, many devices, if it can also 
+get or access a Trusted Component signer key that is trusted by the TEEs. 
+This makes TAMs high value targets. An advanced, persistent threat actor 
+could potentially compromise a TAM without impacting its certificate or 
+raising concern from the TAM's operator.
+
+To mitigate this threat, TEE Agents have several options as listed below but 
+not limit to these for detecting and mitigating a compromised TAM:
 
 1. Apply an ACL to the TAM, indicating which TAs it is permitted to install.
```

Comment:
```suggestion
1. Apply an ACL to the TAM key, limiting which TAs the TAM is permitted to install.
```

Reply:
Hmm, need clarification on this.

An ACL is to include a list of TA by name or ID such that a remote TAM will not be able to install new TAs, achieving limited damage. Such an ACL may be implemented in REE, for example, applying to the SUIT manifest processing time, which contains TA IDs and TAMs that will be used. In such an implementation, a TAM key isn't accessible. However, it can still limit a compromised TAM's damage via ACL over TAM URL.

An ACL could be implemented and checked inside TEE Agent too in validating a TAM. In this case, TEEP Agent needs to expose an API interface for some REE applications to update TA list in an ACL, which differs from trust anchors that can be preloaded and largely remain static.

Reply:
> An ACL is to include a list of TA by name or ID

Disagree.   As Brendan mentioned, it may be a list of install paths, or a list of trusted component signer keys, or whatever else, it need not be a list of TAs.

> Such an ACL may be implemented in REE

Disagree, it should be implemented in the TEE.  The REE may have no visibility at all, and the device is not even required to have a REE.

> In this case, TEEP Agent needs to expose an API interface for some REE applications to update TA list in an ACL

I don't think the document should suggest that an REE application can have any control over TEE data.  The whole point of the TEE is to defend against anything malicious in the REE.

Reply:
Removed implication of REE option, and let us move on.